core/controllers/reader_test.py

```diff
@@ -231,8 +231,7 @@ def test_give_feedback_handler(self) -> None:
 
 
 class ExplorationPretestsUnitTest(test_utils.GenericTestBase):
-    """Test the handler for initialising exploration with
-    state_classifier_mapping.
+    """Test the handler for retrieving exploration pre-tests.
```

Comment:
Note: I fixed this, since the existing docstring was incorrect (copy-paste error).